android-auto/build.gradle.kts

```
@@ -21,19 +21,25 @@ android {
 }
 
 dependencies {
-    api("com.mapbox.navigation:android:2.0.0-rc.4")
-    api("com.mapbox.search:mapbox-search-android:1.0.0-beta.19")
+    api("com.mapbox.navigation:android:2.0.0-rc.7")
```

```suggestion
    api("com.mapbox.navigation:android:2.0.0-rc.8")
```

https://github.com/mapbox/mapbox-navigation-android/releases/tag/v2.0.0-rc.8 💃

---

Oh yes, thanks for noticing!